resources/views/checkout/partials/sections/success/summary.blade.php

```diff
@@ -14,23 +14,23 @@
 </div>
 
 <div class="flex flex-col !mt-0">
-    <template v-for="segment in checkout.totals.total_segments" v-if="segment.title">
-        <div v-if="segment.code === 'tax'" class="flex justify-between order-first mt-2">
+    <template v-if="order.grand_total">
```

Comment:
Do we need this template tag? 
There's another if check down lower, this would also hide the tax and the shipping if this is not truthy